bambi/backends/stan.py

```diff
@@ -69,9 +69,13 @@ def build(self, spec, reset=True):
         self.X['N'] = n_cases
 
         def _sanitize_name(name):
-            ''' Stan only allows alphanumeric chars and underscore, so replace
-            all invalid chars with '_' and track for later re-substitution. '''
-            clean = re.sub('[^a-zA-Z0-9\_]+', '_', name)
+            ''' Stan only allows alphanumeric chars and underscore, and
+            variable names must begin with a letter. So replace all invalid
+            chars with '_', prepend with 'b_', and track for later
+            re-substitution. '''
+            if name in self._original_names:
+                return name
+            clean = 'b_' + re.sub('[^a-zA-Z0-9\_]+', '_', name)
```

Comment:
I thought we were going to drop the 'u_' and 'b_' prefixes? Or do you want to leave them in for now and handle that separately? I'm fine either way.